CHANGELOG.md

```diff
@@ -39,6 +39,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     - all Base Data Fixtures were removed
     - the data are created either in database migrations or in Demo Data Fixtures
 
+### [shopsys/coding-standards]
+#### Added
+- [#308 - Sniff for forgotten dumps](https://github.com/shopsys/shopsys/pull/308)
+    - added support for checking standards of file types twig, html, js
```

Comment:
I'm not sure... did we add support for html? Because I see only `.twig`, `.html.twig`, `.js` in source code